Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:latest
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm ci
+
+COPY . .
+
+RUN npm run build
+
+EXPOSE 5173
```

Comment:
why do we need this? `EXPOSE` doesn't "expose" the port: https://docs.docker.com/reference/dockerfile/#expose but is more of a doc. Is that the intent?

Reply:
Yea for documentation since vite uses a different port than Nuxt.